cmd/pj-rehearse/rehearse.go

```diff
@@ -0,0 +1,588 @@
+package main
```

Comment:
Moving the code to a different file makes the PR really hard to review and also to me it doesn't make any sense to do it since it is still in the same package. Perhaps this should live in `pkg/rehearse`